src/components/GlobalPreferences/GlobalPreferences.js

```diff
@@ -248,8 +227,6 @@ Close.propTypes = {
 
 function AnimatedGlobalPreferences(props) {
   const { sectionIndex, subsection, handleNavigation } = useGlobalPreferences({
-    locator: props.locator,
-    onScreenChange: props.onScreenChange,
     wrapper: props.wrapper,
```

Comment:
Perhaps I'm missing something but I can't see how `useGlobalPreferences` is making use of `wrapper`?